plugins/messagelayer/parameters.go

```diff
@@ -20,7 +20,7 @@ type ParametersDefinition struct {
 	}
 
 	// TangleTimeWindow defines the time window in which the node considers itself as synced according to TangleTime.
-	TangleTimeWindow time.Duration `default:"2m" usage:"the time window in which the node considers itself as synced according to TangleTime"`
```

Comment:
This should only be changed for our deployment. In general I think 2min is okay

Reply:
Done